index.bs

```diff
@@ -151,16 +167,24 @@ spec:url; type:dfn; text:scheme
 
         1. [=Dispatch=] |event| to |successorWindow|.
 
-        1. If |event|'s [=PortalActivateEvent/predecessor browsing context=] is not null, then
-            [=queue a task=] from the [=portal task source=] to the
-            [=event loop=] associated with |predecessorBrowsingContext| to
-            resolve |promise| with undefined.
+        1. Let |adoptedPredecessorElement| be |event|'s [=PortalActivateEvent/adopted predecessor element=].
+
+        1. If |adoptedPredecessorElement| is not null, then:
+
+            1. Set |adoptedPredecessorElement|'s [=just-adopted flag=] to false.
+
+            1. If |element| [=may have a guest browsing context|may not have a guest browsing context=] and its [=guest browsing context=] is not null, then [=discard a browsing context|discard=] it.
```

Comment:
The use of discard here vs. close below warrants a note, I think.

Reply:
Done; wdyt?